### DISCUSSION TITLE
Does ``wrapping`` property really work?

### DISCUSSION THREAD

**Discussion by kusab85** [laravel/nova-issues#5112] [Q&A]
There's a ``Field`` property called ``wrapping``

```php
    /**
     * Indicates if the field should allow its whitespace to be wrapped.
     *
     * @var bool
     */
    public $wrapping = false;
```

Shouldn't setting ``wrapping`` to ``true`` change Field's view on Resource's Index page?

![obraz](https://user-images.githubusercontent.com/53086618/203534583-1335c7c4-e1a4-440f-84fb-0613b14f4288.png)

**Comment by crynobone** [comment]
The `props` are only used to apply classes on `<th>` and `<td>`

**Reply by kusab85** [reply]
So the only way to get long text lines wrapped is to configure field ``asHtml()`` and ``displayUsing(fn ($value) => htmlentities($value))``.

```php
Text::make(__('Item name'), 'name')
    ->sortable()
    ->withMeta(['wrapping' => true])
    ->asHtml()
    ->displayUsing(fn ($name) => htmlentities($name)),
```

But that's a bummer because the field doesn't trigger click action then.
